src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+    operate(operation, value) {
+      result = operation(result, value);
```

Comment:
Ensure that the `operation` parameter is a valid function before calling it. You can add a check to verify that `operation` is a function to prevent runtime errors.